src/app.js

```diff
@@ -1,12 +1,82 @@
-import axios from 'axios';
+import axios from "axios";
 
-// Async - await Full data
-async function fetchData() {
+const countryList = document.getElementById('country-list');
+// const countryImage = document.getElementById('country-flag')
+
+//1. data fetchen
+
+console.log("test tekst")
+
+// const usernameInputField = document.getElementById('input-search');
+
+async function fetchDataCountries() {
+    const URI = 'https://restcountries.com/v2/all'
```

Comment:
Variabelen in camelCase schrijven
```suggestion
    const uri = 'https://restcountries.com/v2/all'
```